.github/workflows/npm-publish.yml

```diff
@@ -3,49 +3,50 @@ name: npm-publish
 on:
   workflow_dispatch:
     inputs:
-      package-name:
-        description: Package name
-        type: choice
-        default: "couper"
-        required: true
-        options:
```

Comment:
Lets add the matrix probs here:

```yaml
strategy:
      matrix:
        pkg: ['couper', '@avenga/couper']
```

and read from `${{ matrix.pkg }}` later on

Reply:
:star_struck: